sdk/maps/azure-maps-examples/src/main/java/com/azure/maps/examples/SearchSample.java

```diff
@@ -24,6 +24,7 @@
 import com.azure.maps.search.models.GeographicEntityType;
 import com.azure.maps.search.models.LatLong;
 import com.azure.maps.search.models.OperatingHoursRange;
+import com.azure.maps.search.models.Polygon;
```

Comment:
This file is completely modified with no particular reason. Cannot commit throwaway / test code into the main branch. Please reset this file back to its original state: https://stackoverflow.com/questions/7147270/hard-reset-of-a-single-file